astrokat/test/test_offline_observe.py

```diff
@@ -36,18 +40,25 @@ def test_two_calib_sim(self):
         cal1 = result.count("0408-65 observed for 30.0 sec")
         cal2 = result.count("1934-638 observed for 30.0 sec")
 
-        self.assertGreaterEqual(cal1 + cal2, 1, "At least one bpcal was observed")
+        self.assertGreaterEqual(
+            cal1 + cal2, 1, "At least one bpcal was observed")
         self.assertLessEqual(cal1 + cal2, 2, "At most 2 bpcals were observed")
 
     def test_drift_targets_sim(self):
+        """Get result and make sure everything ran properly."""
         execute_observe_main("test_obs/drift-targets-sim.yaml")
 
-        # get result and make sure everything ran properly
         result = LoggedTelescope.user_logger_stream.getvalue()
-        self.assertIn("Single run through observation target list", result, "Single run")
+        self.assertIn(
+            "Single run through observation target list",
+            result,
+            "Single run")
         self.assertIn("0408-65 observed for 180.0 sec", result)
         self.assertIn("1934-638 observed for 180.0 sec", result)
-        self.assertEqual(result.count("Drift_scan observation for"), 2, "two drift scans")
+        self.assertEqual(
+            result.count("Drift_scan observation for"),
+            2,
+            "two drift scans")
```

Comment:
Here as well:

`
return func(*args, **keywargs)
File "/home/kat/workspace/946_linting_with_flake8_astrokat/astrokat/test/test_offline_observe.py", line 61, in test_drift_targets_sim
`

`
"two drift scans")
AssertionError: 4 != 2 : two drift scans
'4 != 2' = '%s != %s' % _common_shorten_repr(4, 2)
'4 != 2 : two drift scans' = self._formatMessage('4 != 2 : two drift scans', '4 != 2')
raise self.failureException('4 != 2 : two drift scans')
`